tests/indexes_/test_search_indexes.py

```diff
@@ -146,20 +133,21 @@ def test_multiple_fields(self):
                         "representation": "double",
                         "type": "number",
                     },
+                    "object_id": {"type": "objectId"},
                 },
             }
             self.assertCountEqual(index_info[index.name]["columns"], index.fields)
             self.assertEqual(index_info[index.name]["options"], expected_options)
-            self.assertAddRemoveIndex(editor, Article, index)
+            editor.remove_index(index=index, model=Article)
```

Comment:
If you look at `schema/tests.py`, you'll see that this sort of clean up should be in a `finally:` block. You can create multiple `with connection.schema_editor() as editor:` blocks so that the entire method isn't indented under a single one.